flareon/src/db.rs

```diff
@@ -175,45 +204,18 @@ impl Iden for &Identifier {
 #[derive(Debug, Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash)]
 pub struct Column {
     name: Identifier,
-    auto_value: bool,
-    unique: bool,
-    null: bool,
 }
```

Comment:
Because these fields are useless. The proc macro generated them, but the proc macro cannot reliably check if a field is auto or null (because it depends on the exact type of the field, and we can't resolve types in proc macros). As about `unique`, it is not useful during runtime (it's only really relevant in migrations).

I want to keep the struct though even though it only contains one field for now, because it's highly likely we'll need to add some field metadata in the future that will affect the behavior in runtime.